AIDevGallery/Samples/SharedCode/GenAIModel.cs

```diff
@@ -84,6 +85,11 @@ private GenAIModel(string modelDir)
         return model;
     }
 
+    public static void InitializeGenAI()
+    {
+        _ogaHandle = new OgaHandle();
```

Comment:
Could we not just auto do this on each GenAIModel instance so it auto handles this both for the gallery and for the generated projects? If we can avoid having devs to have to call this, we should.

Reply:
No, it doesn't work. If you add this to GenAIModel, ogahandle will be disposed when the model is disposed and next time it will crash. In the gallery it was added to Program.cs.
It should be initialized and disposed only once in the app